Constitution.txt

```diff
@@ -1,490 +1,264 @@
+7: Judgment in Cases of impeachment shall not extend further than to removal from Office, and disqualification to hold and enjoy any Office of honor, Trust or Profit under these United States: but the Party convicted shall nevertheless be liable and subject to Indictment, Trial, Judgment and Punishment, according to Law.
 
 Section 4
-1: The Times, Places and Manner of holding Elections for Senators and Representatives, shall be prescribed in each State by the Legislature thereof; but the Congress may at any time by Law make or alter such Regulations, except as to the Places of chusing Senators.
+1: The Times, Places and Manner of holding Elections for Representatives, and for deciding the Appointments of Senators, shall be prescribed in each State by the Constitution thereof, but such Elections must occur every second Year from the date on which the State joins the Union.
 
-2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December,5 unless they shall by Law appoint a different Day.
+2: The Congress shall assemble at least once in every Year, and such Meeting shall be on the first Monday in December, unless they shall by Law appoint a different Day. Neither house shall assemble for a period of more than one Hundred days in a year, except as may be necessary in time of emergency which threatens the immediate physical safety of the Citizens of any State.
 
 Section 5
 1: Each House shall be the Judge of the Elections, Returns and Qualifications of its own Members, and a Majority of each shall constitute a Quorum to do Business; but a smaller Number may adjourn from day to day, and may be authorized to compel the Attendance of absent Members, in such Manner, and under such Penalties as each House may provide.
 
-2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behaviour, and, with the Concurrence of two thirds, expel a Member.
+2: Each House may determine the Rules of its Proceedings, punish its Members for disorderly Behavior, and, with the Concurrence of two thirds, expel a Member, who shall when so expelled be ineligible to return until one full term of Representation has elapsed.
 
-3: Each House shall keep a Journal of its Proceedings, and from time to time publish the same, excepting such Parts as may in their Judgment require Secrecy; and the Yeas and Nays of the Members of either House on any question shall, at the Desire of one fifth of those Present, be entered on the Journal.
+3: Each House shall keep a Journal of its Proceedings, and within 3 months publish the same, excepting such Parts as may in their Judgement require Secrecy. All matters which are so excepted shall be published within 15 Years of the Proceedings, though this may be extended a further 6 years in times of declared War.
```

Comment:
I place time limits on the publishing of Proceedings, because modern technology has allowed for improvements in documentation, and the goings-on of the Congress should be public. Any secrecy needs to be specific and limited. Roughly one generation is plenty of time for the Congress to prepare for a secret to be made public, and any national security concerns allayed.